usr/share/rear/restore/NETFS/default/51_set_capabilities.sh

```diff
@@ -0,0 +1,11 @@
+# restore capabilities if capabilities are in the backup
+	if test -s $VAR_DIR/recovery/capabilities ; then
+		Log "Restoring Capabilities."
+		if which setcap >/dev/null 2>&1 ; then
+			while read file cap ; do
+				setcap $cap ${TARGET_FS_ROOT}/${file}	
+			done < <(cat $VAR_DIR/recovery/capabilities | sed 's/=//')
+		else
+			Log "setcap binary not found."
```

Comment:
As you added `setcap` and `getcap` to the `REQUIRED_PROGS` array remove the `if which` check as it is unneeded (rest assure the test for REQUIRED_PROGS works fine).